src/main/java/com/webforj/samples/views/fields/maskedtextfield/MaskedTextFieldRestoreView.java

```diff
@@ -0,0 +1,32 @@
+package com.webforj.samples.views.fields.maskedtextfield;
+
+import com.webforj.component.Composite;
+import com.webforj.component.button.Button;
+import com.webforj.component.field.MaskedTextField;
+import com.webforj.component.layout.flexlayout.FlexAlignment;
+import com.webforj.component.layout.flexlayout.FlexDirection;
+import com.webforj.component.layout.flexlayout.FlexJustifyContent;
+import com.webforj.component.layout.flexlayout.FlexLayout;
+import com.webforj.router.annotation.Route;
+
+@Route
+public class MaskedTextFieldRestoreView extends Composite<FlexLayout>{
```

Comment:
1. Use a meaningful mask instead of the default one.  
2. Align everything to the left, and make sure the input and button are horizontally aligned.  
3. Use the primary theme for the button.  
4. Add a label to the input field.